src/main/java/Main.java

```diff
@@ -56,18 +51,41 @@ public static void main(String[] args) throws Exception {
                     System.out.println("your role");
                     String inputRole = scanner.nextLine();
                     UserRole createRole = UserRole.valueOf(inputRole.toUpperCase());
-                    new RegisterCommand().register(createUserName, createPassword, createName, createSurname, createRole);
-                    break;
-
-                case CommandConstants.EXIT_CODE:
+                    new RegisterCommand().
```

Comment:
1. Why do you have new line? Row can't finish with `.`
2. Look at other commands execution and make one style everywhere